tox.ini

```diff
@@ -5,13 +5,10 @@ envlist = py{27,36}
 # into the virtualenvs tox creates.
 skipsdist = true
 
-# `ansible-review-standards/standards.py` fails to execute on Python 2.7, due to
-# how its import system works.
 [testenv]
-commands =
-    ansible-galaxy install git+https://github.com/ANXS/postgresql.git,,ANXS.postgresql
-    molecule test
 deps =
     ansible
     docker-py
     molecule
+commands =
+    molecule test
```

Comment:
I've re-ordered this section to list "deps" before "commands" so that the code better reflects the order in which tox acts: it installs dependencies before running commands.